tests/fixture/testnet/local/node.go

```diff
@@ -308,7 +320,32 @@ func (n *LocalNode) IsHealthy(ctx context.Context) (bool, error) {
 		}
 	}
 	// Assume all other errors are not recoverable
-	return false, err
+	return false, fmt.Errorf("failed to query node health: %w", err)
+}
+
+// WaitForHealthy blocks until IsHealthy returns true or an error (including context timeout) is observed.
+func (n *LocalNode) WaitForHealthy(ctx context.Context) error {
```

Comment:
Can we remove this function from the `Node` interface and instead make this a function that takes in any implementation of the `Node` interface?